packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -540,6 +763,61 @@ class InteractiveViewer extends StatefulWidget {
     return closestOverall;
   }
 
+  // Convert an axis aligned Quad to a Rect.
+  //
+  // All Rects must axis aligned.
```

Comment:
Which "rects" is this referring to?

Reply:
All of them.  I'll change it to this:

"All instances of Rect are axis aligned by definition."

Meanwhile, Quads are not necessarily axis aligned.